sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -114,7 +115,8 @@ EnvironmentLogLevelListener::GetLogListener()
                   << Azure::DateTime(std::chrono::system_clock::now())
                          .ToString(
                              DateTime::DateFormat::Rfc3339, DateTime::TimeFractionFormat::AllDigits)
-                  << "] " << LogLevelToConsoleString(level) << " : " << message << std::endl;
+                  << " T: " << std::this_thread::get_id() << "] " << LogLevelToConsoleString(level)
```

Comment:
I'm not comfortable making that level of surgery to the websockets pull request, especially since most of these changes are needed to make the CI pipeline pass.